maven-plugin/src/main/java/org/openmrs/maven/plugins/AddExclusion.java

```diff
@@ -51,22 +54,23 @@ public void executeTask() throws MojoExecutionException, MojoFailureException {
             throw new MojoFailureException("Invalid distro file");
         }
 
-        Artifact distroArtifact = originalDistroProperties.getParentArtifact();
-        if (StringUtils.isNotBlank(distroArtifact.getArtifactId()) && StringUtils.isNotBlank(distroArtifact.getGroupId()) && StringUtils.isNotBlank(distroArtifact.getVersion())) {
-            DistroProperties distroProperties = distroHelper.resolveParentArtifact(distroArtifact, new File(distro).getParentFile(), originalDistroProperties, null);
```

Comment:
In my testing, this was actually not getting the full set of parent properties.  This code also does not resolve full ancestors (eg. if the parent has a parent).  The change below intends to fix this.